### PR TITLE
ch(#13): Setup unit testing circle ci and coveralls badges

### LINKED ISSUE #13: Setup Unit Testing, circle ci, and coveralls badges

### PR DESCRIPTION
**Pull request by munezeromicha**
**Description**

This pull request introduces the following setup to improve our project's code quality and continuous integration process:

**Unit Testing Setup:**

- Unit test Implemented is Jest.
- Configured the testing environment to run tests consistently across different environments.

**CircleCI Integration:**

- Set up CircleCI for continuous integration.
- Configured CircleCI to run the unit tests on every commit and pull request.
- Ensured the build pipeline is stable and reliable.

**Coveralls Integration:**

- Integrated Coveralls to track code coverage.
- Added the Coveralls badge to the README file to display the current coverage status.
- Configured the project to automatically upload coverage reports to Coveralls after each build.

### CODE REVIEW

**Comment by ncutixavier** [comment]
- Your test failed, how can I merge the setup test when tests are failing
- This PR is not ready

**Comment by munezeromicha** [comment]
*Yes! am working on it!*

**Review by fredshema — APPROVED** [review]

**Review by Salim-54 — APPROVED** [review]

File: tailwind.config.js
Comment (fredshema) [inline tailwind.config.js:1]: This needs to be in `jest.config.js` not `tailwind.config.js`

File: package.json [hunk 1]
```diff
@@ -1,48 +1,76 @@
...
+    "scripts": {
+        "start": "expo start",
+        "android": "expo start --android",
+        "ios": "expo start --ios",
+        "web": "expo start --web",
+        "start:wsl": "REACT_NATIVE_PACKAGER_HOSTNAME=$(netsh.exe interface ip show address 'Wi-Fi' | grep 'IP Address' | sed -r 's/^.*IP Address:\\W*//') expo start",
+        "test": "jest --coverage",
+        "test:watch": "jest --watch",
+        "test:coverage": "jest --coverage",
+        "coverage": "nyc npm test && nyc report --reporter=text-lcov | coveralls"
```
Comment (fredshema) [inline package.json]: Specify one command to generate coverage reports

File: app/+html.tsx
Comment (fredshema) [inline app/+html.tsx:1]: Remove this file for web, we will not need a web version for the app

**Review by fredshema — CHANGES_REQUESTED** [review]

**Review by fredshema — APPROVED** [review]

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**